contrib/local-environment/docker-compose-keycloak.yaml

```diff
@@ -43,7 +43,7 @@ services:
 
   keycloak:
     container_name: keycloak
-    image: jboss/keycloak:10.0.2
+    image: quay.io/keycloak/keycloak:10.0.2
```

Comment:
...

Eventhough this is just a test file. We should update to the latest keycloak version which is 24.

For that we will need to update the keycloak realm and user file as well. 

@kvanzuijlen do you know how to do it or should i add to this PR?

Reply:
I have no experience with keycloak, so it may be better if you did that.